test/functional/test_framework/test_framework.py

```diff
@@ -139,15 +144,18 @@ def main(self):
             self.options.tmpdir = tempfile.mkdtemp(prefix="test")
         self._start_logging()
 
-        success = False
+        success = TestStatus.FAILED
 
         try:
             self.setup_chain()
             self.setup_network()
             self.run_test()
-            success = True
+            success = TestStatus.PASSED
         except JSONRPCException as e:
             self.log.exception("JSONRPC error")
+        except SkipTest as e:
+            self.log.warning("Test Skipped: %s" % e.message)
```

Comment:
OK, I see. I thought the log.info below was also logging something similar. This is fine.